src/MusicStore/Controllers/ManageController.cs

```diff
@@ -119,25 +116,25 @@ public IActionResult AddPhoneNumber()
         // POST: /Manage/DisableTwoFactorAuthentication
         [HttpPost]
         [ValidateAntiForgeryToken]
-        public async Task<IActionResult> DisableTwoFactorAuthentication()
+        public async Task<IActionResult> DisableTwoFactorAuthentication(CancellationToken cancellationToken)
```

Comment:
@Praburaj 
They were originally requestAborted, but it has been changed to cancellationToken based on @pranavkm 's suggestion. 

@pranavkm  Actually, I agree with @Praburaj. I understand cancellationToken should be a common name for that type, but in this case requestAborted should more understandable to people who are not familiar to that type. Let me know your opinion on this. I can revert them back